### PR TITLE
Problem: (fix #1828)abci info endpoint returns uncommitted state

### LINKED ISSUE #1828: Problem: abci info endpoint returns uncommitted state
https://docs.tendermint.com/master/spec/abci/apps.html#info-connection

```
QueryState should be set to the latest DeliverTxState at the end of every Commit, ie. after the full block has been processed and the state committed to disk. Otherwise it should never be modified.
```

It currently returns uncommitted state, which could potentially cause data inconsistency.

### PR DESCRIPTION
**Pull request by linfeng-crypto**
solution: get info from storage.

### CODE REVIEW

File: chain-abci/src/app/mod.rs [hunk 1]
```diff
@@ -68,7 +69,9 @@ impl<T: EnclaveProxy> abci::Application for ChainNodeApp<T> {
         let mut resp = ResponseInfo::new();
         resp.app_version = chain_core::APP_VERSION;
         resp.version = get_version();
-        if let Some(app_state) = &self.last_state {
+        if let Some(raw) = chain_storage::get_last_app_state(&kv_store!(self)) {
```
Comment (yihuang) [inline chain-abci/src/app/mod.rs:72]:
```suggestion
        if let Some(raw) = chain_storage::get_last_app_state(&self.storage) {
```
`kv_store!` is the storage combined with in-memory buffer.

**Comment by tomtau** [comment]
bors retry

**Pull request merged**